src/main/steps/applicant/task-list/content.test.ts

```diff
@@ -60,15 +60,19 @@ const cyContent = {
   iWantTo: 'Rwyf eisiau ...',
   hyperlinks: [
     {
-      label: 'Add a legal representative-welsh',
+      label: 'Ychwanegu cynrychiolydd cyfreithiol',
       link: '/applicant/add-legal-representative',
     },
     {
-      label: 'Find my local court-welsh',
+      label: 'Dileu cynrychiolydd cyfreithiol',
+      link: '/applicant/remove-legal-representative/start',
```

Comment:
why is the link for find my local court is mapped to /applicant/remove-legal-representative/start

Reply:
Hugh added this, Looks like he added this for.. test case validity. I could see that in content.ts.. sonali added respective URL.